crates/bevy_pbr/src/render/light.rs

```diff
@@ -229,7 +233,9 @@ impl ShadowPipelineKey {
         let primitive_topology_bits = ((primitive_topology as u32)
             & Self::PRIMITIVE_TOPOLOGY_MASK_BITS)
             << Self::PRIMITIVE_TOPOLOGY_SHIFT_BITS;
-        Self::from_bits(primitive_topology_bits).unwrap()
+        Self::from_bits(primitive_topology_bits).unwrap_or_else(|| {
+            panic!("Could not create `ShadowPipelineKey` from {primitive_topology_bits} bits.")
```

Comment:
Does Display on `primitive_topology_bits` work? I want to say that Debug works but perhaps double-check?

Reply:
`primitive_topology_bits` is a `u32` so `{}` is fine. I thought wrong. :)